app/templates/components/n-times.hbs

```diff
@@ -0,0 +1,3 @@
+{{#each numOfTimes as |time|}}
```

Comment:
Instead of using a computed property, can you use `{{#each (range 1 times) as |number|}}`